src/js/profile/mobile/widget/DropdownMenu.js

```diff
@@ -469,6 +469,7 @@
 					target = event.target;
 
 				if (ns.getConfig("keyboardSupport")) {
+					console.log("onFocus");
```

Comment:
Remove unnecessary console

Reply:
I know. Thank's :) . This is still in progress. I had to rebase this commit.